### ISSUE TITLE
SPI documentation

### ISSUE THREAD

**Issue by mossmann** [dominicgs/GreatFET-experimental#124]
Write a page on the wiki explaining our SPI functions and how they can be used from the command-line tool. Point to the command-line tool as sample code for the SPI Python API. See the Bus Pirate SPI guide for inspiration.

**Comment by travisgoodspeed** [comment]
What is the proper way to set the SPI mode in a GreatFET host script?  The default seems to be to set the MOSI output on the falling edge of the clock, but a target of mine needs the data to be sampled on that edge and asserted on the rising edge.

Answering my own question:  The `gf.spi.transmit()` has an optional `spi_mode` parameter in the Git repo, but not in the public version of GreatFET installed by `pip3`.

**Comment by ktemkin** [comment]
Yep -- the `spi_mode` parameter isn't in a release yet -- but will be soon. You'll need to use master to be able to set the SPI mode.

[I keep putting off this release to add "one more feature", which I really shouldn't do. :)]
